docker-compose.yml

```diff
@@ -23,18 +36,31 @@ services:
       dockerfile: client_mobile.dockerfile
     volumes:
       - client_data:/data/client
-  #apollo:
-  #  build:
-  #    context: ./database
-  #    dockerfile: Dockerfile
-  #  ports:
-  #    - "8082:8082"
-  #  depends_on:
-  #    - db
-  #db:
-  #  image: postgres
-  #  volumes:
-  #    - postgres_data:/data/db
+  apollo:
+    build:
+      context: ./database
+      dockerfile: Dockerfile
+    restart: always
+    ports:
+      - "8082:8082"
+    depends_on:
+      - db
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_DB: ${POSTGRES_DB}
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    networks:
+      - server_network
+    ports:
+      - "5432:5432"
+networks:
+  server_network:
+    driver: bridge
```

Comment:
Isn't bridge the default network driver reachable by each services ?
Did you mean to create a custom one ?

Reference :
https://docs.docker.com/compose/networking/#specify-custom-networks

Reply:
the goal was to network only containers supposed to communicate together, but after reading i should test if it's still working only by default, good one thx

Reply:
Ok, conversation resolved